pom.xml

```diff
@@ -28,12 +28,46 @@
 		<module>schema-to-pojo-gwt</module>
 		<module>schema-to-pojo-integration-tests</module>
 	</modules>
+	<properties>
```

Comment:
Make sure we update the version of all the modules in this lib so that we can pull it in the repo.